Zend/tests/stack_limit/stack_limit_001.phpt

```diff
@@ -4,6 +4,7 @@ Stack limit 001 - Stack limit checks with max_allowed_stack_size detection
 <?php
 if (!function_exists('zend_test_zend_call_stack_get')) die("skip zend_test_zend_call_stack_get() is not available");
 if (getenv('SKIP_MSAN')) die("skip msan requires a considerably higher zend.reserved_stack_size due to instrumentation");
+if (PHP_OS_FAMILY === 'Darwin' && strpos(php_uname('m'), 'aarch64') !== false) die("skip Broken on Apple Silicon");
```

Comment:
Should this be `skip` or `xfail`?

Reply:
I don't personally find `xfail` super useful right now, because it only emits a warning when the test passes unexpectedly. As such, the test result is essentially completely ignored (because nobody looks at warnings). I think now that we have `--FLAKY--` it may make sense to make `xfail` error when the test passes to indicate that the `xfail` flag may be removed.